tm1638/tm1638.go

```diff
@@ -0,0 +1,137 @@
+/*
+TM1638 is an LED Controller
+*/
+package tm1638
+
+import (
+	"machine"
+	"time"
+)
+
+// Device wraps the pins of the TM1638.
+type Device struct {
+	/* STB - When this pin is "LO" chip accepting transmission */
+	strobe machine.Pin
+	/* CLK - DIO pin reads data at the rising edge and outputs at the falling edge */
+	clock machine.Pin
+	/* DIO - This pin outputs/inputs serial data */
+	data machine.Pin
+}
+
+const (
+	addressAutoIncrement = 0x40
+	fixedAddress         = 0x44
+	baseAddress          = 0xC0
+	maxAddress           = 0x0F
+	readKeys             = 0x42
+	brightness           = 0x88
+	maxBrightness        = 0x07
+	zeroBrightness       = 0x80
```

Comment:
Some (but not all) of these constants are commands. Can you prefix them in some ways? For example, `cmdBrightness` would be a lot clearer to me.

Reply:
I will fix.